baseframe/templates/baseframe/bootstrap3/autoform.html.jinja2

```diff
@@ -1,4 +1,4 @@
-{% extends "baseframe/bootstrap3/formlayout.html.jinja2" %}
+{% extends "formlayout.html" %}
```

Comment:
This should not refer to `formlayout.html` since that file is deprecated.

Reply:
Hasjob has its own formlayout.html